src/app/pages/TopicPage/constants.js

```diff
@@ -0,0 +1,16 @@
+export const VISUAL_STYLE = {
```

Comment:
Just had a thought... should all new files be TypeScript instead of JavaScript?

Reply:
Yes. The current coding standard is that all new components should be developed in TypeScript instead of JavaScript.

Reply:
Because this is essentially a refactor pulling out the constants from the Curation promo index I think it's fine as JS